apps/main/src/loan/components/PageLlamaMarkets/cells/RateCell.tsx

```diff
@@ -21,6 +22,7 @@ export const RateCell = ({ market, type }: { market: LlamaMarket; type: RateType
   const { rewards, type: marketType } = market
   const rewardsAction = getRewardsAction(marketType, type)
   const poolRewards = rewards.filter(({ action }) => action == rewardsAction)
+  const isMobile = useMediaQuery((t) => t.breakpoints.down('tablet'))
```

Comment:
Not too important, but having a hook in `curve-ui-kit/utils` would be great. Something like: `const { isMobile } = useBreakpoints()`. It's a tiny bit easier than having to remember `t.breakpoints.down` or `.up`.

Reply:
yeah I agree it's about time for that